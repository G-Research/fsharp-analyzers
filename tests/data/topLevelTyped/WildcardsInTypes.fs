module M

let f (g: _ -> unit) : unit = ignore<int -> unit> g; ()

let h (i: int, j: _) : int = ignore<string> j; 0

let k : _ -> int = printfn "meh"; fun (a: int) -> a + 1
