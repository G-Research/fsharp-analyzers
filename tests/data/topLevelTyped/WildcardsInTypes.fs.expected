GRA-UNTYPEDAPI-001 | Warning | (3,4 - 3,5) | `f` is not private, not fully typed and could be made private to file.
Add type annotation for parameter `g`.
GRA-UNTYPEDAPI-001 | Warning | (5,4 - 5,5) | `h` is not private, not fully typed and could be made private to file.
Add type annotation for parameter `j`.
GRA-UNTYPEDAPI-001 | Warning | (7,4 - 7,5) | `k` is not private, not fully typed and could be made private to file.
Specify the return type.
